src/test/java/com/um/inventoryservice/PresentationLayer/OrderControllerIntegrationTest.java

```diff
@@ -133,6 +134,27 @@ void getStockOrdersByQuantity() {
                     assertThat(orders).extracting("quantity").contains(6);
                 });
     }
+
+    @Test
+    void getStockOrdersByQuantityByClient() {
+        Publisher<Order> setup = orderRepository.deleteAll().thenMany(orderRepository.save(order));
+
+        StepVerifier
+                .create(setup)
+                .expectNextCount(1)
+                .verifyComplete();
+
+        webTestClient.get().uri("/orders/297445493/quantity")
+                .exchange()
+                .expectStatus().isOk()
+                .expectBodyList(StockOrderDTO.class)
+                .hasSize(1)
+                .consumeWith(stockList -> {
+                    List<StockOrderDTO> orders = stockList.getResponseBody();
+                    assertThat(orders).extracting("quantity").contains(6);
```

Comment:
what is extracting exactly?

Reply:
Retrieving the value of the indicated attribute.